.flake8

```diff
@@ -1,24 +1,20 @@
 [flake8]
-max-line-length = 88
-ignore = E501, E203, W503
+ignore =
```

Comment:
Cleaned up as best I can through trial and error.

Reply:
We should at a later time leverage in-line excludes where we really need them and eventually remove these.

Reply:
I think we're always going to keep E501 for what it's worth, as black is not strict about its line wrapping and flake8 is (which is why we don't tell flake8 about our line length).